CONTRIBUTING.rst

```diff
@@ -11,9 +14,11 @@ In no particular order, examples include:
 - Documentation improvements
 - Tutorials
 
-No contribution is too small. Although, contributions can be too big, so let's discuss via the `dev email list`_ OR `an issue`_.
+No contribution is too small. Although, contributions can be too big, so let's
+discuss via `Matrix`_ OR via `an issue`_.
 
-.. _`dev email list` : https://groups.google.com/forum/#!forum/projectmesa-dev
+.. _`Mesa discussions`: https://github.com/projectmesa/mesa/discussions
+.. _`Matrix`: https://matrix.to/#/!CAAdpmuJhdNJkxurrp:matrix.org?via=matrix.org
```

Comment:
This one is fine to be public, and better with human-readable name.

Reply:
I created the Human Readable name #ProjectMesa:matrix.org

Reply:
>90% of the room name is in the format of `#project-mesa:matrix.org`, with very, very few outliers, e.g. `#AuroraSupport:auroraoss.com`.

This is how Element name their rooms:
- `#element-web:matrix.org`
- `#element-android:matrix.org`
- `#matrix:matrix.org`

and so on.

Reply:
updated to #project-mesa:matrix.org